### ISSUE TITLE
[Epic] EZR Zero Silent Failure Epic

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#93398]
## Benefit to the Veteran
As a Veteran, I want [ ], so that [ ].

As a Veteran, I want to know when my EZR has failed so that I know the VA will follow up with me.

As a team member, I would like to know when an EZR has failed so that I can make sure HEC reaches out to the Veteran and there are no silent failures and no data is missed.

## Hypothesis or Bet
**If** _we make this change_ **then** _we expect this to happen_.

If we send silent failure emails and update our monitoring tagging, we will be in compliance with Zero Silent Failure

## OKR
_Which Objective / Key Result does this epic push forward?_

## Background
- _What details are necessary for understanding the specific work or request tracked by this issue?_
- _What are we doing within the scope of this ticket?_
- _Is there anything specifically out of scope of this ticket?_
- _Is there anything that must be done before this ticket? (link ticket as dependency)_
- _Are there any external dependencies?_

## Resources
- Product or Initiative brief links 
- Design links
- Screenshots
- Reference docs/tickets where we have this kind of work documented
- Etc.
---
## Acceptance Criteria/Definition of Done
-  _What will be created or happen as a result of this story?_

---

```[tasklist]
### Tasks
- [ ] Research
- [ ] Design
- [ ] Team review & signoff
- [ ] UI update
- [ ] Team review & signoff
- [ ] BE update
- [ ] QA
- [ ] Team review & signoff
- [ ] End to end testing
- [ ] Release Plan
- [ ] Cypress test suite
- [ ] Team review & signoff
- [ ] Launch to Prod
- [ ] Update Source of Truth
```

---
## How to configure this issue
- [ ] **Labeled with Team** (`1010-team`)
- [ ] **Labeled with Form** (`1010-ez`, `1010-ezr`, `1010-cg`)
- [ ] **Labeled with Project** (`facilities-cg`, `registration`,  `toxic-exposure`, etc.)
